web/themes/custom/contribtracker/package.json

```diff
@@ -3,11 +3,15 @@
     "build": "gulp",
     "lint": "CI=true gulp lint",
     "vr": "percy exec -- cypress run",
-    "cypress-open": "cypress open --browser chrome",
-    "cypress-run": "cypress run --browser chrome",
-    "cypress:run:without-video": "CYPRESS_VIDEO=false cypress run"
+    "cypress-open": "cypress open --env version=prod --browser chrome",
+    "cypress-run": "cypress run --env version=prod --browser chrome",
```

Comment:
Didn't we get rid of `version=prod` previously? @zeshanziya, do you remember?

Reply:
@hussainweb Yes, we removed this. I kind of remember  the same.